tests/constantinople/create2/test_create_returndata.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
Could we take this opportunity to rename folder `create2` to ` eip1014_create2`?

Reply:
Yes, also need to enable coverage